eng/mgmt/gulpfile.js

```diff
@@ -39,9 +39,10 @@ gulp.task('default', function() {
     console.log("\tThe version of AutoRest. E.g. 2.0.9, or the location of AutoRest repo, e.g. E:\\repo\\autorest");
 
     console.log("--autorest-java");
-    console.log("\tPath to an autorest.java generator to pass as a --use argument to AutoRest.");
+    console.log("\tOption#1: Path to an autorest.java generator to pass as a --use argument to AutoRest.");
+    console.log("\t\tOption#2: The version of AutoRest.Java. E.g. 2.0.9. You can also pass latest or preview.");
```

Comment:
I think option should use same number of `'\t'`

Reply:
How about use another parameter `--autorest-java-version` rather than use same parameter?